src/main/java/seedu/address/ui/SettingsDisplay.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.logic.PetManager;
+import seedu.address.model.ReadOnlyPomodoro;
+
+public class SettingsDisplay extends UiPart<Region> {
+    private static final String FXML = "SettingsDisplay.fxml";
+
+    private ReadOnlyPomodoro pomodoro;
```

Comment:
We may refactor the code to remove Pomodoro and petManager out of SettingsDisplay in the future. The values can perhaps be passed in from ModelManager.

Reply:
ok, will refactor that in the future!